src/aligners/tests.rs

```diff
@@ -11,11 +11,21 @@ fn test1() {
     let cm = AffineCost::new_affine(1, 2, 1);
     let tmp: NW<AffineCost<2>> = NW { cm: cm.clone() };
     let tmp2: DiagonalTransition<AffineCost<2>> = DiagonalTransition::new(cm);
-    for i in 1..20 {
+    let start = 1;
+    let stop = 500;
+    let step = 5;
+    let mut len = start;
+    'main: loop {
         // Test section
-        let (ref a, ref b, ref _alphabet, _stats) = setup(i, 0.3);
-        println!("{i}");
-        print!("s1 == {}\ns2 == {}\n", to_string(&a), to_string(&b));
-        assert_eq!(tmp.align(a, b).0, tmp2.align(a, b).0);
+        for k in 0..=5 {
+            let (ref a, ref b, ref _alphabet, _stats) = setup(len, 0.2 * k as f32);
+            println!("{len}");
+            print!("s1 == {}\ns2 == {}\n", to_string(&a), to_string(&b));
+            assert_eq!(tmp.align(a, b).0, tmp2.align(a, b).0);
```

Comment:
You can add extra argument to the assert similar to format. It's nicer to out the debug prints in there